apps/emqx/src/emqx_schema.erl

```diff
@@ -529,14 +583,16 @@ fields("force_gc") ->
     [ {"enable",
        sc(boolean(),
           #{ default => true
-           })}
+            })}
     , {"count",
        sc(range(0, inf),
-          #{ default => 16000
+          #{ default => 16000,
+             desc =>  "GC the process after this many received messages"
            })}
     , {"bytes",
        sc(bytesize(),
-          #{ default => "16MB"
+          #{ default => "16MB",
+             desc => "GC the process after how much bytes passed through"
```

Comment:
nit: Phrasing is a bit awkward. Perhaps something like: GC the process after specified number of bytes have passed through.